Modules/Core/Common/include/itkArray.h

```diff
@@ -104,7 +104,7 @@ class ITK_TEMPLATE_EXPORT Array : public vnl_vector<TValue>
   {
     this->m_LetArrayManageMemory = true;
     this->SetSize(r.GetSize());
-    for (SizeValueType i = 0; i < r.GetSize(); i++)
+    for (SizeValueType i = 0; i < r.GetSize(); ++i)
```

Comment:
Ahhh, `++i` looks so strange!

Reply:
> Ahhh, `++i` looks so strange!

@dzenanz Are you serious? We could introduce a function, `itk::Increment(i)`...  😸